src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml

```diff
@@ -17,10 +17,16 @@
     <Page.Resources>
         <!--  TODO: Figure out what we want to do here for filtering/grouping and where  -->
         <!--  https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.data.collectionviewsource  -->
-        <!--<CollectionViewSource
-            x:Name="ItemsCVS"
+        <CollectionViewSource
+            x:Name="GroupedItemsCVS"
             IsSourceGrouped="True"
-            Source="{x:Bind ViewModel.Items, Mode=OneWay}" />-->
+            ItemsPath="Items"
+            Source="{x:Bind ViewModel.Groups, Mode=OneWay}" />
+
+        <CollectionViewSource
+            x:Name="UngroupedItemsCVS"
+            IsSourceGrouped="False"
+            Source="{x:Bind ViewModel.FilteredItems, Mode=OneWay}" />
```

Comment:
This is where the issue lies, is ideally we don't duplicate the list of filtered/grouped items and the UI/containers between them.

This is where CVS in WinUI only supporting grouping is an issue and the ACV in the Toolkit only filtering, there's not the unification of sorting, filtering, and grouping. (FYI @marcpems) https://github.com/microsoft/microsoft-ui-xaml/issues/4307 and https://github.com/microsoft/WinUI-Gallery/issues/1625

It was a good excuse for me to publish my old testing repo for ItemsRepeater: https://github.com/michael-hawker/ItemsRepeaterTests - probably don't need the nested ScrollViewer, but that could be a path to nest ItemsRepeaters, and use a template selector for the ungrouped bucket. Of course we've run into issues trying to use ItemsRepeater.